shell/platform/embedder/embedder.h

```diff
@@ -2601,9 +2615,18 @@ FlutterEngineResult FlutterEngineAddView(FLUTTER_API_SYMBOL(FlutterEngine)
 //------------------------------------------------------------------------------
 /// @brief      Removes a view.
 ///
-///             This is an asynchronous operation. The view's resources must not
-///             be cleaned up until the |remove_view_callback| is invoked with
-///             a |removed| value of `true`.
+///             This operation notifies the engine to deallocate resources and
+///             inform Dart about the removal. Finally, it invokes
+///             |info.remove_view_callback| with whether the operation is
+///             successful.
+///
+///             This operation is asynchronous. The embedder should not
+///             deallocate resources until the |callback| is invoked.
+///
+///             Do not use for implicit views, which are never removed
+///             throughout the lifetime of the app.  Removing
+///             |kFlutterImplicitViewId| or an non-existent view ID will fail,
```

Comment:
Folks copy this file into their own projects, they won't have this `kFlutterImplicitViewId` constant. I would use a little more generic language here:

```suggestion
///             the implicit view ID or an non-existent view ID will fail,
```